content/docs/reference/_index.md

```diff
@@ -0,0 +1,18 @@
++++
+title="Reference"
+weight=2
```

Comment:
Because our docs/site is more centered toward newcomers I would think this would come last in the menu. Similar to how it is on the index (docs) page. Changing the weight to `4` would make it consistent (aka last on the menu).

Reply:
totally agree. i was wondering how the order was determined! i'll fix that